### DISCUSSION TITLE
How to implement dark mode toggle?

### DISCUSSION THREAD

**Discussion by hexdecimal16** [wix/react-native-ui-lib#1789] [Q&A] [answered]
I want to provide dark mode toggle in the app as a setting. Currently I am able to switch between dark mode and light mode by changing it from android settings. How to also provide it as an toggle option in settings?

**Comment by MuhammedKpln** [comment, marked as answer]
Using schemes.

You'll need a custom theme colors for each theme

```
Colors.loadSchemes({
  light: {
    screenBG: 'transparent',
    textColor: Colors.grey10,
    moonOrSun: Colors.yellow30,
    mountainForeground: Colors.green30,
    mountainBackground: Colors.green50
  },
  dark: {
    screenBG: Colors.grey10,
    textColor: Colors.white,
    moonOrSun: Colors.grey80,
    mountainForeground: Colors.violet10,
    mountainBackground: Colors.violet20
  }
});
```

Then afterwards you can use 
```
Colors.setScheme('dark/light')

```

For toggling between dark mode

**Reply by ethanshar** [reply]
Make sure you use these modifiers (`screenBG`, `textColor`, etc...)  on the components you wish to react to dark mode
